### PR TITLE
Fix a bug when disable trusty and create /dev/sdax failed.

### PR DESCRIPTION
**Pull request by tanminger**
Jira: None.
Test: Test it in KBL NUC.

Signed-off-by: Ming Tan <ming.tan@intel.com>

### CODE REVIEW

**Comment by android-ia-srv** [comment]
Commit message is verified for commit id:  https://github.com/android-ia/bootable-live-installer/pull/35/commits/3de9f764c4e88af6d8566a8f1bd9c94bdf37b5fc <br>

File: initrd/scripts/2-install [hunk 1]
```diff
@@ -155,6 +155,9 @@ create_one_partition()
 	sgdisk -c $num:$partition_label /dev/$device 1> /dev/null
 
 	start=$(( end + 1 ))
+	sleep 1
```
Comment (harishkrupo) [inline initrd/scripts/2-install]: What is the problem here? Is it that the new partitions created are not being reflected?

Comment (tanminger) [reply initrd/scripts/2-install]: After a new partition is created, the /dev/sdax maybe not be created in some times.
Add the 'mdev -s' to hope helpful for it.

Comment (tanminger) [reply initrd/scripts/2-install]: I have tested the partprobe at before, but seems does not work.

Comment (harishkrupo) [reply initrd/scripts/2-install]: Oh, okay.

**Comment by tanminger** [comment]
I just update this patch to make it more stable.

**Comment by tanminger** [comment]
Move to https://github.com/projectceladon/bootable-live-installer.

**Pull request closed without merging**